### DISCUSSION TITLE
Multiple Action Button is showing bellow not inline

### DISCUSSION THREAD

**Discussion by nafiur** [Power-Components/livewire-powergrid#1307] [Q&A] [answered]
Multiple Action Button is showing bellow not inline

![image](https://github.com/Power-Components/livewire-powergrid/assets/4471794/4c582ff0-243b-498a-a06d-d01b64e115a7)
![Screenshot_16](https://github.com/Power-Components/livewire-powergrid/assets/4471794/d32a8b22-51dd-4d3b-a1d2-ba9f9f73caac)

Bootstrap Theme. How to solve this pls help me

**Comment by abhaypithadiya** [comment]
Hey!

Publish the Powergrid view file using:

`php artisan vendor:publish --tag=livewire-powergrid-views`

Go to resources/views/vendor/livewire-powergrid/components/row.blade.php on line 14 before render actions, update the div classes to something like this: 

```
        <div class="d-flex w-full justify-content-center align-items-center"> <----- This line here
            <!-- Render Action -->

            @if (filled(data_get($row, 'actions')) && $column->isAction)
                @foreach (data_get($row, 'actions') as $key => $action)
                    <div wire:key="action-{{ $row->id }}-{{ $key }}">
                        {!! $action !!}
                    </div>
                @endforeach
            @endif
        </div>
```

Hope this helps!

**Comment by nafiur** [comment, marked as answer]
@abhaypithadiya  Thanks For Your Suggestion. In this file line no. 44 to 52 i change like this 

        <div class="pg-actions d-flex w-full justify-content-center align-items-center">
            @if (filled(data_get($row, 'actions')) && $column->isAction)
            @foreach (data_get($row, 'actions') as $key => $action)
            <div wire:key="action-{{ data_get($row, $primaryKey) }}-{{ $key }}" style="margin-right: 5px">
                {!! $action !!}
            </div>
            @endforeach
            @endif
        </div>

and its working.


![Screenshot_18](https://github.com/Power-Components/livewire-powergrid/assets/4471794/d40b2af0-657c-4aa5-8ecb-d26e36ab677b)
